runtime/vm/jvminit.c

```diff
@@ -4218,6 +4218,18 @@ processVMArgsFromFirstToLast(J9JavaVM * vm)
 		} else if (flightRecorder < noFlightRecorder) {
 			vm->extendedRuntimeFlags2 &= ~(UDATA)J9_EXTENDED_RUNTIME2_JFR_ENABLED;
 		}
+
+		IDATA jfrOptionIndex = FIND_AND_CONSUME_VMARG(OPTIONAL_LIST_MATCH_USING_EQUALS, VMOPT_XXSTARTFLIGHTRECORDING, NULL);
+		if (jfrOptionIndex >= 0) {
+			char* jfrOptionBuffer = NULL;
+			GET_OPTION_VALUE(jfrOptionIndex, '=', &jfrOptionBuffer);
+			if(jfrOptionBuffer == NULL) {
```

Comment:
See https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#formatting-conditions for how conditions should be formatted